src/client/datascience/ipywidgets/cdnWidgetScriptSourceProvider.ts

```diff
@@ -222,9 +222,6 @@ export class CDNWidgetScriptSourceProvider implements IWidgetScriptSourceProvide
         let success = false;
         while (retryCount > 0 && !success) {
             try {
-                if (await this.fs.localFileExists(tempFile.filePath)) {
```

Comment:
Not quite sure that I get the fix? localFileExists hangs or fails on Windows somehow?

Reply:
it was a file permission thing, I guess creating a new file wasn't giving permissions

Reply:
Well given the usage here, seem like an ok change. Worst case we get some temp files on disk.